manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -35,7 +35,7 @@ gateways:
     rollingMaxSurge: 100%
     rollingMaxUnavailable: 25%
     autoscaleEnabled: true
-    autoscaleMin: 1
+    autoscaleMin: 2
```

Comment:
See my comment on https://github.com/istio/istio/pull/25873#pullrequestreview-455854430. I do not think this is a change we want to be making

Reply:
Looking at all of the issues and replies related to `PDB` starting from `1.3`, Most of the folks recommend `minReplicas=2` and it solves their problems. https://github.com/istio/istio/issues/12602#issuecomment-663169633 https://github.com/istio/istio/issues/12602#issuecomment-701038118
https://github.com/istio/istio/issues/12602#issuecomment-661487599

How about making `minReplicas` configurable so cluster with resource restriction will set it to lower values?

Reply:
As mentioned in the comment, I am perfectly happy with allowing it to be higher, making an HA profile, etc. I am not, personally, ok with the default being greater than 1

Reply:
Put another way - I would rather have no PDB than 2 replicas default. Others may disagree with me though

Reply:
@sdake WDYT?

Reply:
agree with @howardjohn for the default profile. It is simple to add a profile and the profile solves the problem you are after solving. I don't see problems with adding profiles that make sense. The directory is here: https://github.com/istio/istio/tree/master/manifests/profiles

As to how the profile behaves, I don't have a strong opinion. But I think changing the default profile in this way will lead to frustration from smaller operators during the evaluation period.

Cheers,
-steve